pkg/functions/function.go

```diff
@@ -394,19 +408,46 @@ func (f Function) Write() (err error) {
 //
 // The runtime data directory .func is created in the function root if
 // necessary.
-func (f Function) Stamp() (err error) {
+func (f Function) Stamp(oo ...stampOption) (err error) {
```

Comment:
Journaling is very much an optional, debugging-focused feature, so I used the option pattern here to retain the default API as the happy-path